Tests/DocsXsd/DocsXsdTest.php

```diff
@@ -0,0 +1,156 @@
+        $this->assertSame(0, $result['exitcode'], 'Exit code does not match 0');
+        $this->assertSame('', $result['stdout'], 'Unexpected output in stdout');
+        $this->assertNotFalse(
+            \strpos($result['stderr'], "Tests/Fixtures/DocsXsd/{$fixtureFile} validates\n"),
+            'Unexpected output in stderr'
+        );
+    }
+
+    /**
+     * Data provider for valid test cases.
+     *
+     * @return array
+     */
+    public function dataValidXsd()
+    {
+        return [
+            'Valid docs example with single standard in the file' => [
+                'fixtureFile' => 'ValidSingleStandard.xml',
+            ],
+            'Valid docs example with multiple standards in the file' => [
+                'fixtureFile' => 'ValidSingleStandard.xml',
+            ],
+            'Valid docs example with multiple attributes' => [
+                'fixtureFile' => 'ValidDocumentationWithAdditionalAttributes.xml',
+            ],
+            'Valid docs example with multiple attributes on the standard element' => [
+                'fixtureFile' => 'ValidStandardWithAdditionalAttributes.xml',
+            ],
+        ];
+    }
+
+    /**
+     * Verify that the invalid XML will throw the correct validation errors
+     *
+     * @dataProvider dataInvalidXsd
+     *
+     * @param string $fixtureFile    The name of the fixture file in the fixture directory.
+     * @param string $expectedStdOut Expected output from the xmllint command.
+     * @param string $expectedStdErr Expected validation error from the xmllint command.
+     *
+     * @return void
+     */
+    public function testInvalidXsd($fixtureFile, $expectedStdOut, $expectedStdErr)
+    {
+        $command = \sprintf(self::COMMAND, $fixtureFile);
+        $result  = $this->executeCliCommand($command);
+
+        $this->assertGreaterThan(0, $result['exitcode'], 'Exit code does not match 0');
+        $this->assertSame($expectedStdOut, $result['stdout'], 'Unexpected output in stdout');
+        $this->assertNotFalse(\strpos($result['stderr'], $expectedStdErr), 'Unexpected output in stderr');
+    }
+
+    /**
+     * Data provider for invalid test cases.
+     *
+     * @return array
+     */
+    public function dataInvalidXsd()
+    {
+        return [
+            'Title attribute too long' => [
+                'fixtureFile'    => 'InvalidTitleTooLong.xml',
+                'expectedStdOut' => '',
+                'expectedStdErr' => "this exceeds the allowed maximum length of '58'",
+            ],
+            'Documentation root element missing' => [
+                'fixtureFile'    => 'InvalidMissingDocumentationRoot.xml',
+                'expectedStdOut' => '',
+                'expectedStdErr' => "Element 'standard': No matching global declaration available for the validation root",
+            ],
+            'Missing title attribute in the documentation root element' => [
+                'fixtureFile'    => 'InvalidMissingTitleAttribute.xml',
```

Comment:
```suggestion
                'fixtureFile'    => 'InvalidMissingDocumentationTitleAttribute.xml',
```

(will also need a file rename)

Reason: clarify that the test fixture is about the `<documentation>` element title, not the `<code>` element which also requires a title.